client/src/components/generic/Dialog.js

```diff
@@ -1,18 +1,27 @@
 import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
 import { Box, Typography, Dialog as MuiDialog} from '@material-ui/core';
 
+const useStyles = makeStyles(() => ({
+  root: {
+    width: '1000px',
```

Comment:
Should this line exist? The `maxWidth` is set to 600px.

Reply:
Yeah, this was really weird and I guess I forgot to come back to it. Didn't work with `width: 100%`. Will look at it again.